driver.go

```diff
@@ -75,7 +75,7 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 		mcnflag.StringFlag{
 			EnvVar: "HETZNER_API_TOKEN",
 			Name:   flagApiToken,
-			Usage:  "Project-specific Hetzner API token",
+			Usage:  "Project-specific Hetzner API token (FOO)",
```

Comment:
Are you sure you wanted to commit that? ^^

Reply:
Nop :-)